docs/installation/manual/configuring_drupal.md

```diff
@@ -0,0 +1,163 @@
+# If Carapace was downloaded, now is the time to enable and set it as well.
+drush -y theme:enable carapace
+drush -y config-set system.theme default carapace
+# After all of this, run updates and rebuild the cache.
+cd /opt/drupal
+sudo -u www-data composer update
+drush -y cr
+```
+
+### Adding a JWT Configuration to Drupal
+
+To allow our installation to talk to other services via Syn, we need to establish a Drupal-side JWT configuration using the keys we generated at that time.
+
+Log onto your site as an administrator at `/user`, then navigate to `/admin/config/system/keys/add`. Some of the settings here are unimportant, but pay close attention to the **Key type**, which should match the key we created earlier (an RSA key), and the **File location**, which should be the ultimate location of the key we created for Syn on the filesystem, `/opt/keys/syn_private.key`.
+
+![Adding a JWT RSA Key](../../assets/adding_a_jwt_rsa_key.png)
+
+Click **Save** to create the key.
+
+Once this key is created, navigate to `/admin/config/system/jwt` to select the key you just created from the list. Note that before the key will show up in the **Private Key** list, you need to select that key's type in the **Algorithm** section, namely `RSASSA-PKCS1-v1_5 using SHA-256 (RS256)`.
+
+![Configuring the JWT RSA Key for Use](../../assets/configuring_the_jwt_rsa_key_for_use.png)
+
+Click **Save configuration** to establish this as the JWT key configuration.
+
+### Configuring Islandora
+
+Navigate to the Islandora core configuration page at `/admin/config/islandora/core` to set up the core configuration to connect to Gemini. Of note here, the **Gemini URL** will need to be established to facilitate the connection to Fedora, and the appropriate **Bundles with Gemini URI pseudo field** types will need to be checked off.
+
+!!! notice
+    Any other Drupal content types you wish to synchronize with Fedora should also be checked off here.
+
+![Configuring Islandora](../../assets/configuring_islandora.png)
+
+### Configuring Islandora IIIF
+
+Navigate to `/admin/config/islandora/iiif` to ensure that Islandora IIIF is pointing to our Cantaloupe server.
+
+![Configuring Islandora IIIF](../../assets/configuring_iiif.png)
+
+Next, configure Openseadragon by navigating to `/admin/config/media/openseadragon` and ensuring everything is set up properly.
+
+![Configuring Openseadragon](../../assets/configuring_openseadragon.png)
+
+### Establishing Flysystem as the Default Download Method
+
+Navigate to `/admin/config/media/file-system` to set the **Default download method** to the one we created in our `settings.php`.
+
+![Configuring Flysystem to Use Fedora](../../assets/configuring_flysystem_to_use_fedora.png)
+
+### Running Feature Migrations
+
+Finally, to get everything up and running, run the Islandora Core Features and Islandora Defaults migrations.
+
+```bash
+cd /opt/drupal
+drush -y -l localhost --userid=1 mim --group=islandora
```

Comment:
Make sure the user you run this as has read permissions on the private key, or else it won't work. (I had to run it as `sudo -u www-data drush -y -l localhost:8000 --userid=1 mim --group=islandora`

Reply:
I ran mine on port 80 so I didn't need the `:8000` in the url param, but otherwise, yes, you need to set `--userid=1` and run it as the apache user.

Reply:
Clarification: the userid flag is to run it as the Drupal Admin user (for Drupal permissions). The `sudo -u www-data` is what runs it as the apache user (for filesystem permissions).